src/test/java/org/folio/rest/impl/WebServiceTest.java

```diff
@@ -57,10 +57,10 @@ static void beforeAll(Vertx vertx, VertxTestContext context) {
   @AfterAll
   static void afterAll(Vertx vertx, VertxTestContext context) {
     webClient.close();
-    vertx.close(context.succeeding(x -> {
+    vertx.close().onComplete(x -> {
       log.debug("afterAll completed");
       context.completeNow();
-    }));
+    });
   }
```

Comment:
The afterAll method is not needed.

The VertxExtension automatically closes vertx: https://vertx.io/docs/vertx-junit5/java/#_closing_and_removal_of_vertx_objects

Closing vertx also closes the webClient that depends on vertx.